data/Configuration.py

```diff
@@ -1,30 +1,33 @@
 import os
+
+
 class Configuration:
     def __init__(self):
         self.authentication_type ="http_signature"
         self.merchantid = "testrest"
         self.run_environment = "CyberSource.Environment.SANDBOX"
-        self.request_json_path = "../../../../cybersource-rest-samples-python/resources/request.json"
+        self.request_json_path = ""
+        # JWT PARAMETERS
         self.key_alias = "testrest"
         self.key_pass = "testrest"
         self.key_file_name = "testrest"
-        self.keys_directory = os.getcwd() + "\\resources\\"
+        self.keys_directory = os.path.join(os.getcwd(), "resources")
+        # HTTP PARAMETERS
         self.merchant_keyid = "08c94330-f618-42a3-b09d-e1e43be5efda"
         self.merchant_secretkey = "yBJxy6LjM2TmcPGu+GaJrHtkke25fPpUX+UY6/L/1tE="
-        self.enable_log = True
+        # CONNECTION TIMEOUT PARAMETER
         self.timeout = 1000
+        # LOG PARAMETERS
+        self.enable_log = True
         self.log_file_name = "cybs"
         self.log_maximum_size = 10487560
-        self.log_directory = os.path.join(os.getcwd(),"Logs")
+        self.log_directory = os.path.join(os.getcwd(), "Logs")
+        # PROXY PARAMETERS
         self.proxy_address = "userproxy.visa.com"
         self.proxy_port = ""
```

Comment:
Replace the proxy address and port with dummy values. Merchant should manually change this if proxy is required.